src/addons/grids/ReactDataGrid.js

```diff
@@ -517,9 +517,14 @@ const ReactDataGrid = React.createClass({
       this.setState({selected: selected});
     }
   },
-
+    
+  // Logic extented to allow for functions to be passed down in column.editable
+  // this allows us to deicde whether we can be edting from a cell level
   canEdit(idx: number): boolean {
     let col = this.getColumn(idx);
```

Comment:
to remove duplication you can put this function into columnUtils.js and create a test for it there, using a fake column with an editable function